sample/build.gradle

```diff
@@ -0,0 +1,65 @@
+apply plugin: 'com.android.application'
+apply plugin: 'kotlin-android'
+apply plugin: 'kotlin-kapt'
+
+repositories {
+    jcenter()
+}
+
+android {
+    compileSdkVersion 28
+
+    defaultConfig {
+        minSdkVersion 21
+        targetSdkVersion 28
+        versionCode 1
+        versionName "1.0.0"
+        applicationId "com.github.terrakok.cicerone.sample"
+    }
+
+    buildTypes {
+        release {
+            minifyEnabled false
+            proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
+        }
+
+        debug {
+        }
+    }
+
+    buildFeatures {
+        viewBinding true
+    }
+}
+
+ext {
+    androidXVersion = "1.2.0"
+    materialVersion = "1.2.1"
+    moxyVersion = "1.7.0"
+    daggerVersion = "2.29.1"
+}
+
+dependencies {
+    // Support libraries
+    implementation "androidx.appcompat:appcompat:$androidXVersion"
+    implementation "com.google.android.material:material:$materialVersion"
+
+    //MVP Moxy
+    implementation "tech.schoolhelper:moxy-x:$moxyVersion"
+    implementation "tech.schoolhelper:moxy-x-androidx:$moxyVersion"
+    kapt "tech.schoolhelper:moxy-x-compiler:$moxyVersion"
+
+    //Cicerone
+    implementation project(':library')
+
+    //DI
+    implementation "com.google.dagger:dagger:$daggerVersion"
+    kapt "com.google.dagger:dagger-compiler:$daggerVersion"
+
+    //Bottom Navigation Bar
+    implementation ('com.ashokvarma.android:bottom-navigation-bar:1.3.0') {
+        exclude group: "com.android.support", module: "design"
+    }
+    implementation "androidx.core:core-ktx:1.3.2"
+    implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version"
```

Comment:
kotlin plugin adds dependency on stdlib automaticaly

Reply:
That one was added by Android Studio. I removed that line.